webpack.common.ts

```diff
@@ -147,52 +164,8 @@ module.exports = {
       STATIC_PREFIX: BASE_PATH,
     }),
     new MonacoWebpackPlugin({
-      languages: [],
-      // features: ['!gotoSymbol'],
-      features: [
-        '!accessibilityHelp',
-        '!anchorSelect',
-        '!bracketMatching',
-        '!caretOperations',
-        '!clipboard',
-        '!codeAction',
-        '!codelens',
-        '!colorDetector',
-        '!contextmenu',
-        '!coreCommands',
-        '!dnd',
-        '!find',
-        '!folding',
-        '!fontZoom',
-        '!format',
-        '!gotoError',
-        '!gotoLine',
-        '!gotoSymbol',
-        '!hover',
-        '!iPadShowKeyboard',
-        '!inPlaceReplace',
-        '!inspectTokens',
-        '!linesOperations',
-        '!links',
-        '!multicursor',
-        '!onTypeRename',
-        '!parameterHints',
-        '!quickCommand',
-        '!quickHelp',
-        '!quickOutline',
-        '!referenceSearch',
-        '!rename',
-        '!smartSelect',
-        '!snippets',
-        '!toggleHighContrast',
-        '!toggleTabFocusMode',
-        '!transpose',
-        '!unusualLineTerminators',
-        '!viewportSemanticTokens',
-        '!wordHighlighter',
-        '!wordOperations',
-        '!wordPartOperations',
-      ],
+      languages: ['json', 'markdown'],
+      filename: '[name].worker.bundle.js',
```

Comment:
Same filename output, and same language settings, for both `webpack.common.ts` and `webpack.vendor.ts`. In the end, we'll still end up with only a single copy -- across the two, separate webpack build steps.

note our build step, with the 2 webpack builds:
`yarn generate && yarn build:vendor && webpack --config webpack.prod.ts`